src/interface/read_config_file.py

```diff
@@ -0,0 +1,108 @@
+import xml.etree.ElementTree as etree
+class ReadConfigFile():
+    
+    def __init__(self, fileconfig): 
+            
+            
+            self.xmld = etree.parse(fileconfig)             
+            self.root = self.xmld.getroot()
+
+        
+
+    def get_output_file(self):
+        """ 
+        Returns the outputfile path, name and extension values from config file
```

Comment:
check the name of your method get_output_file and in the documentation you have outputfile in one word that is not correct